sst/SimengCoreWrapper.cc

```diff
@@ -0,0 +1,236 @@
+// DO NOT MOVE FROM TOP OF FILE
+#include <sst/core/sst_config.h>
```

Comment:
This fails the format check but we need to put this at the top of the file apparently. Not sure what the fix will be.